Makefile

```diff
@@ -1,42 +1,50 @@
 DOCKER_REGISTRY ?= gcr.io/linkerd-io
 REPO = $(DOCKER_REGISTRY)/proxy-init
 
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help: ## Show this help
+	@echo 'Info: For integration test using KinD, run: make kind-load && make integration-test'
```

Comment:
Awesome! I believe `make kind-load integration-test` is enough, no?

Reply:
ah yes, you are right